pkg/services/alerting/notifiers/base.go

```diff
@@ -18,7 +18,7 @@ type NotifierBase struct {
 	Name                  string
 	Type                  string
 	UID                   string
-	IsDeault              bool
+	IsDefault             bool
```

Comment:
@marefr @bergquist Do you know if it's safe to rename a field on `NotifierBase`? I'm just wondering if it could affect external entities, e.g. through notifiers being sent.

Reply:
Based on search of usage should be safe

Reply:
Great, thanks for clarifying that @marefr.